single-account-single-cluster-multi-env/30.eks/30.cluster/main.tf

```diff
@@ -39,7 +44,62 @@ module "eks" {
   ))
 
   enable_cluster_creator_admin_permissions = "true"
-  authentication_mode                      = "API_AND_CONFIG_MAP"
+  authentication_mode                      = try(var.cluster_config.authentication_mode, "API_AND_CONFIG_MAP")
+
+  bootstrap_self_managed_addons = "false"
+
+  cluster_addons = merge(
```

Comment:
Why we've added the addon into the EKS module and not the EKS blueprints addons module?